tools/benchmark/src/index.ts

```diff
@@ -39,14 +39,14 @@ export {
 	CustomBenchmarkArguments,
 } from "./Configuration";
 export { benchmark, benchmarkMemory, IMemoryTestObject, MemoryTestObjectProps } from "./mocha";
-export { prettyNumber, geometricMean, Stats } from "./ReporterUtilities";
+export { prettyNumber, geometricMean } from "./RunnerUtilities";
 export { BenchmarkReporter } from "./Reporter";
+export { Phase, runBenchmark } from "./runBenchmark";
 export {
-	BenchmarkData,
 	BenchmarkError,
+	BenchmarkData,
```

Comment:
Nit: this used to be in alphabetical order with BenchmarkData before BenchmarkError, let's restore that :).